app/models/concerns/text_sanitizer.rb

```diff
@@ -22,4 +22,16 @@ def sanitize_text_only(text)
     return nil if text.blank?
     Loofah.fragment(text).scrub!(:prune).text(encode_special_chars: false)
   end
+
+  def sanitize_keywords(kws, strict)
+    Array(kws).map { |kw| sanitize_keyword(kw, kw.length, strict) }.uniq.reject(&:blank?)
+  end
+
+  def sanitize_keyword(kw, max_length, strict)
```

Comment:
should we rename this since it isn't used for `keywords` anymore?

Reply:
Well... it is still used for `episode.keyword_xid` in addition to categories.  Which... do we still use that at all?  That was an Adzerk thing, right?

Reply:
We do not use that for anything - it was supposed to be way to do episode targeting in adzerk/kevel, but guid has always worked better - I'll enter an issue to remove it, actually (doesn't need to be in this PR!)

Reply:
https://github.com/PRX/feeder.prx.org/issues/1032

Reply:
Cool - renamed the helper method.  So for that ticket, can just nuke the field.